docs/api/version-history.md

```diff
@@ -50,6 +50,16 @@ keywords: "API, Docker, rcli, REST, documentation"
   if they are not set.
 * `GET /info` now omits the `KernelMemory` and `KernelMemoryTCP` if they are not
   supported by the host or host's configuration (if cgroups v2 are in use).
+* `POST /containers/create` for Windows containers now rejects non-empty values in
+  `HostConfig.Resources.Devices.PathInContainer` and
+  `HostConfig.Resources.Devices.CgroupPermissions` that were previously silently ignored.
+  This change is not versioned, and affects all API versions if the daemon has this patch.
```

Comment:
It doesn't appear that the described behavior is aligned with the implementation. `func (*Daemon) createSpec` is only called in one place: `func (*Daemon) containerStart`. Therefore `PathInContainer` and `CgroupPermissions` will not be checked until one attempts to start the container.

Could this API change be versioned such that for API requests with version <= 1.41, the existing accept-but-ignore behavior is retained?